examples/react-dashboard/src/App.js

```diff
@@ -44,17 +48,14 @@ function Example() {
 
   const isDatasetSelected = (cardId, dataKey) => {
     if (
-      myPresence?.selectedDataset?.cardId === cardId &&
-      myPresence?.selectedDataset?.dataKey === dataKey
+      myPresence.selectedDataset?.cardId === cardId &&
+      myPresence.selectedDataset?.dataKey === dataKey
     ) {
       return true;
     }
 
-    for (const other of others.toArray()) {
-      if (
-        other.presence?.selectedDataset?.cardId === cardId &&
-        other.presence?.selectedDataset?.dataKey === dataKey
-      ) {
+    for (const other of others) {
+      if (other[1]?.cardId === cardId && other[1]?.dataKey === dataKey) {
```

Comment:
I personally find the following syntax a little bit more readable:

```suggestion
    for (const [_, selectedDataset] of others) {
      if (selectedDataset?.cardId === cardId && selectedDataset?.dataKey === dataKey) {
```

Reply:
great suggestion, gonna implement